src/components/BlogCard.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Card, Heading, Paragraph } from "./styles"
+import { MdDelete } from "react-icons/md"
+
+const BlogCard = ({ title, author, content, id }) => {
+  return (
+    <Card>
+      <div>
+        <Heading>
+          <b>author: {author}</b>
```

Comment:
![image](https://user-images.githubusercontent.com/54318933/177763030-86c8510e-639a-4c4e-b1f8-220877e112b7.png)


https://www.w3schools.com/tags/tag_b.asp

Reply:
okay. i actually didnt know i used this